docs/core-concepts/atomic-swap-htlc.md

```diff
@@ -0,0 +1,146 @@
+---
+id: atomic-swap-htlc
+title: Atomic Swaps using HTLCs
+sidebar_label: Atomic Swaps using HTLCs
+---
+
+This documentation section provides an introduction to atomic swaps, hash time locked contracts (HTLCs) and how these concepts are used in COMIT.
+Atomic Swaps and HTLCs are core concepts of the COMIT protocol.
+
+<iframe 
+    src="https://docs.google.com/presentation/d/e/2PACX-1vRjJeto_eQaZ8k5Ct00UzceHaSs-uIuvPmTFV6jP9SP6N-EKmct8H4cfN3xa2h-RFs8Gm4Gia41Je7m/embed?start=true&loop=false&delayms=1000" 
+    frameborder="0" 
+    width="801" 
+    height="480" 
+    allowfullscreen="true"
+    mozallowfullscreen="true" 
+    webkitallowfullscreen="true">
+</iframe>
+
+This slide shows an overview of how atomics swaps using HTLCs work in COMIT, more details can be found in the sections below.
+
+## Atomic swap
+
+An Atomic Swap is an exchange of two digital assets between two parties in which both parties are guaranteed to either receive the other asset OR retain their original asset i.e., an atomic swap guarantees that neither party is able to take both assets.
+
+Atomicity of the swap can be achieved by [HTLC](#hash-time-locked-contract-htlc)s that serve as a programmable escrow, or other concepts as explained in the [Bitcoin Wiki](https://en.bitcoin.it/wiki/Atomic_swap).
+
+## Hash Time Locked Contract (HTLC)
+
+HTLC stands for Hash Time Locked Contract. An HTLC is a script or smart contract that locks a digital asset over time.
+HTLCs require a ledger that provides a time lock.
+
+Both parties are required to have the same knowledge of all parameters for the HTLCs on both ledgers before they can start.
```

Comment:
What are some examples of these parameters?

Reply:
It doesn't have to be tied strictly to our implementation, but knowing a high-level interface would be helpful in understanding the concepts I think.